flake.nix

```diff
@@ -16,7 +16,7 @@
       url = "github:haskell/haskell-language-server";
       flake = false;
     };
-    cardano-node.url = "github:intersectmbo/cardano-node/8.11.0-pre";
+    cardano-node.url = "github:intersectmbo/cardano-node/9.0.0";
     mithril.url = "github:input-output-hk/mithril/2418.1";
```

Comment:
Should also update mithril to `2428.0` as this adds support for cardano node 9.0.0: https://github.com/input-output-hk/mithril/releases/tag/2428.0